src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -51,31 +51,57 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
     isActive,
   } = props;
 
+  // Automatically select filter when one filter group with one option
+  const filterEntries = Object.entries(subjectMeta.filters);
```

Comment:
Seems like in most cases where we're using `Object.entries`, we're only really using the property values, so we could make most of this code a little nicer by using `Object.values` instead as we don't need to access the values in the entries using `[1]` (not very readable).

Would do this for the code in `initialFormValues` too.